index.html

```diff
@@ -1,276 +1,240 @@
-  <title>Patrick Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/port.css">
+    <title>Patrick Portfolio</title>
 </head>
-<body>
-  <!--header-->
- <div class="bk_container">
-  <div class="port_container">
-    <header class="header">
-      <nav class="navbar">
-        <a href="#" class="logo">
-          <img src="./images/Portfolio_Logo.png" alt="My Logo"/></a>
-         <a>
-          <img src="./images/Enabled.png" alt="Menu"/>
-        </a>
-      </nav>
-    </header>
-   <!--main section--> 
-    <div id="main_section" class="grid" >
-     <div class="container">
-      <div class="h1">I’m Patrick Ndimukulaga Glad to see you!</div>
-      <p class="p">
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact
-        me.
-      </p>
-
-      <div>
-        <a href="#" class="media_p">LET'S CONNECT</a>
-        <ul class="media_ul">
-          <li class="media_li">
-            <a href="#"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
-          </li>
-          <li class="media_li">
-            <a href="#"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
-          </li>
-          <li class="media_li">
-            <a href="#"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
-          </li>
-          <li class="media_li">
-            <a href="#"><img src="./images/Github-icon.svg" alt="Github icon"/></a>
-          </li>
-          <li class="media_li"><a href="#"><img src="./images/Vector.svg" alt="Vector icon"/></a>
-          </li>
-        </ul>
-      </div>
-      </div>
-    </div>
-    
-    <section class="simi">
-      <div class="title grid">
-       
-        <div>
-          <img src="./images/Snapshoot_Portfolio.png" alt="portfolio"/>
-        </div>
-        <div class="title-text">
-          <div class="t-text">
-            <div class="port">
-              <h4>Tonic</h4>
-
-              <ul class="ul">
-                <li class="first-child">CANOPY</li>
-                <li>Back End Dev</li>
-                <li>2015</li>
-              </ul>
+  <body>
+    <div id="bk_container">
+        <header class="navbar">
+            <img src="./images/Portfolio_Logo.png" alt="My Logo" class="logo"/>
+            <nav class="nav">
+                <ul>
+                    <li><a href="#" class="menu_li" >Portfolio</a></li>
+                    <li><a href="#" class="menu_li">About</a></li>
+                    <li><a href="#" class="menu_li">Contact</a></li>
+                </ul> 
+            </nav>
+            <img src="./images/Enabled.png" alt="menu" class="menu-icon"/>
+        </header>
```

Comment:
DESKTOP VERSION:

- [ ] Kindly make sure to add space in the header as per the Figma design. The list (Portfolio, About, Contact) is to the right end. Make sure that there is space on the right side.

Figma design:
<img width="612" alt="Screenshot 2022-05-05 at 1 38 15 AM" src="https://user-images.githubusercontent.com/70269507/166841576-dca5b590-b3ff-4ef8-840b-b8e4a288fa24.png">

Your design:
<img width="1547" alt="Screenshot 2022-05-05 at 2 30 32 AM" src="https://user-images.githubusercontent.com/70269507/166841596-f161de7a-3882-48b6-92bd-b2d641e63e1c.png">